src/app/sections/send-coins/send-coins.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+
+import { CurrenciesService } from '../../services/currencies';
+
+@Component({
+  selector: 'send-coins-section',
+  templateUrl: './send-coins.component.html',
+  styleUrls: ['./send-coins.component.scss'],
+  providers: [ CurrenciesService ],
+})
+export class SendCoinsSection implements OnInit {
+
+  currencies: object;
+  errorMessage: string;
+
+  constructor(private currenciesService: CurrenciesService) { }
+
+  ngOnInit() {
+    this.getCurrencies()
+  }
+
+  getCurrencies() {
+    this.currenciesService.getCurrencies()
+      .subscribe(
+        currencies => {
+          // @TODO check data for correct format and display view error if wrong
```

Comment:
Don't forget these checks, lets wait until we have the actual data coming back from changelly though.